migrations/Version202605011800001488_taoAdvancedSearch.php

```
@@ -0,0 +1,77 @@
+        IndexerInterface::DELIVERIES_INDEX,
+        IndexerInterface::GROUPS_INDEX,
+        IndexerInterface::ASSETS_INDEX,
+        IndexerInterface::TEST_TAKERS_INDEX,
+    ];
+
+    public function getDescription(): string
+    {
+        return 'Put Elasticsearch nested mapping for custom metadata field `attributes` (key, type, value, raw_value) on resource indices (items, tests, deliveries, groups, assets, test-takers)';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)